Classes/OpenidService.php

```diff
@@ -221,6 +223,10 @@ public function getUser()
             }
         } elseif (!empty($this->loginData['uident_openid'])) {
             $this->sendOpenIDRequest($this->loginData['uident_openid']);
+        } elseif (!empty(GeneralUtility::_POST('openid_url'))) {
```

Comment:
I am not that happy with this "raw check" either. Just tryed to copy the already used style of this extension. See function 'processLoginData' for example. Same check used.